src/test/resources/epub3/files/content-document-xhtml/aria-roles-section-valid.xhtml

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>	 
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<title>DPUB-ARIA roles allowed on section</title>
```

Comment:
some roles have required owned elements, so the test case is only "valid" since the schema do not check these relationships entirely.

I suggest we add the required context / owned elements everywhere needed to make the test cases theoretically valid (and consequently more future-proof).